### ISSUE TITLE
Grayscale images not detected with OpenFace

### ISSUE THREAD

**Issue by abdou31** [TadasBaltrusaitis/OpenFace#791]
**Describe the bug**
I can say that I have found a bug in OpenFace, I have tested `OpenFaceoffline.exe` on some grayscale images, but I didn't get any .csv output.
Do you have any idea about this?

NB: images are from ibug dataset

**Screenshots**
![044-000028](https://user-images.githubusercontent.com/19480228/67058298-75f44380-f154-11e9-922d-88c1973e854a.jpg)

![044-000036](https://user-images.githubusercontent.com/19480228/67058302-7b518e00-f154-11e9-8cea-bbcdc5160ffb.jpg)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10 x64
 - Version [e.g. 22] 2.2.0

**Comment by lilianabrandao** [comment]
Hello abdou31,
Did you find the solution for this problem? 
I'm facing the same issue... 
Thanks

**Comment by lilianabrandao** [comment]
Hi @abdou31 ,
Did you find the solution for this problem?
I'm facing the same issue...
Thanks

**Comment by abdou31** [comment]
No, I have eliminated those images from the dataset because some of thrm are not detected and some other are detected with negative values

**Comment by lilianabrandao** [comment]
Thanks @abdou31 !

**Comment by TimChaouoft** [comment]
Hi... This is the same problem for me.. I tried analyzing the grayscale image with "FeatureExtraction", but the confidence is low though..
